group_vars/rhcs.yml.sample

```diff
@@ -852,3 +853,4 @@ alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alert
 
 #container_exec_cmd:
 #docker: false
+
```

Comment:
@dsavineau , this new line is generated running the script, do you think we can manually remove it?

Reply:
That's because you added a new line in `roles/ceph-defaults/defaults/main.yml` so it's automatically added in the {all,rhcs}.yml.sample files.